aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -40,8 +40,17 @@ void neg_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+void erfinv_kernel_cuda(TensorIterator& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND_HALF(iter.dtype(), "erfinv_cuda", [&]() {
```

Comment:
_AND_HALF?

Reply:
Currently it's generated  for FLOAT, DOUBLE and HALF